utils/runtime-fuzzer/src/utils.rs

```diff
@@ -32,17 +34,21 @@ pub fn min_unstructured_input_size() -> usize {
     generators.unstructured_size_hint()
 }
 
-pub(crate) fn default_generator_set(test_input_id: String) -> ExtrinsicGeneratorSet {
-    const UPLOAD_PROGRAM_CALLS: usize = 10;
-    const SEND_MESSAGE_CALLS: usize = 15;
-    const SEND_REPLY_CALLS: usize = 1;
+const UPLOAD_PROGRAM_CALLS: usize = 10;
```

Comment:
I'd still consider separate mod for constants in fuzzer

Reply:
that seems necessary now, yeah

Reply:
_fuzzer related comments_
resolving, a new PR for fuzzer changes will be opened.